src/features/abbreviation/abbreviations.json

```diff
@@ -0,0 +1,1753 @@
+{
+	"{}": "{$CURSOR}",
+	"{}_": "{$CURSOR}_",
+	"{{}}": "⦃$CURSOR⦄",
+	"[]": "[$CURSOR]",
+	"[]_": "[$CURSOR]_",
+	"[[]]": "⟦$CURSOR⟧",
+	"<>": "⟨$CURSOR⟩",
+	"()": "($CURSOR)",
+	"()_": "($CURSOR)_",
+	"([])'": "⟮$CURSOR⟯",
+	"f<>": "‹$CURSOR›",
+	"f<<>>": "«$CURSOR»",
```

Comment:
Given https://github.com/leanprover/vscode-lean/pull/249 should we do something like this?
```suggestion
	"f<<>>": "«$CURSOR»",
	"[--]": "⁅$CURSOR⁆",
```

Reply:
Yes! However, I think this should be added as resolution in the merge commit.

Reply:
I just rebased my changes.